ONSdigital/rm-tools
# Remove ref to survey_name and data in collex loader

### Context:
As a result of: https://github.com/ONSdigital/rm-collection-exercise-service/pull/113

There is no need to load the collection service with survey names thus this PR aims to remove all these from the corresponding `.csv` files that get loaded in.

### How to review

- Check that in the `.csvs` survey name column and data associated with it is removed across the board.
- Update the `setup_data.sh` to navigate to this branch in acceptance tests so it can pick up the changes on this branch and then run the `make setup` in `acceptance tests` to verify data being loaded has the column `survey_name` removed.
- Run AT's and check that they pass the usual.
